shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -429,17 +590,14 @@ - (void)testCalculateMultitaskingAdjustment {
   CGRect viewOrigFrame = CGRectMake(0, 0, 320, screenHeight - 40);
   CGRect convertedViewFrame = CGRectMake(20, 20, 320, screenHeight - 40);
   CGRect keyboardFrame = CGRectMake(20, screenHeight - 320, screenWidth, 300);
-
-  id mockView = OCMClassMock([UIView class]);
-  OCMStub([mockView frame]).andReturn(viewOrigFrame);
-  OCMStub([mockView convertRect:viewOrigFrame toCoordinateSpace:[OCMArg any]])
-      .andReturn(convertedViewFrame);
+  id mockView = [self setupMockMainScreenAndView:viewControllerMock
+                                       viewFrame:viewOrigFrame
+                                  convertedFrame:convertedViewFrame];
```

Comment:
Line 585 can be removed since you're using `setupMockMainScreenAndView`
```
585  OCMStub([viewControllerMock mainScreenIfViewLoaded]).andReturn(UIScreen.mainScreen);
```